patches/chromium/fix_add_method_which_disables_headless_mode_on_native_widget.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Cezary Kulakowski <cezary@openfin.co>
+Date: Mon, 22 Jul 2024 16:23:13 +0200
+Subject: fix: add method which disables headless mode on native widget
+
+We need this method as we create window in headless mode and we
+switch it back to normal mode only after inital paint is done in
+order to get some events like WebContents.beginFrameSubscription.
+If we don't set `is_headless_` to false then some child windows
+e.g. autofill popups will be created in headless mode leading to
+ui problems (like dissapearing popup during typing in html's
+input list.
```

Comment:
Same here - is this an issue upstream would ever face @CezaryKulakowski? Can it be upstreamed?

Reply:
Unfortunately, I doubt it can be upstreamed, because we are abusing Chromium Widget headless mode for enforcing painting it before we show the window (as it is stated in the original refactor commit):
```
  // We initialize the window in headless mode to allow painting before it is
  // shown, but we don't want the headless behavior of allowing the window to be
  // placed unconstrained.
```

Reply:
Is this non upstreamable patch a blocker?

Reply:
I don't think this is a blocker, if you looked into it and it's not tenable to upstream, that's alright, but we wanted to at least consider it